### DISCUSSION TITLE
failed to request video translation

### DISCUSSION THREAD

**Discussion by underoakerr** [ilyhalight/voice-over-translation#221] [Q&A]
не переводит ничего, и висит эта ошибка failed to request video translation 
что с этим делать?

**Comment by underoakerr** [comment]
и еще маленькая проблемка, каждые 1-2 минуты, повышается виключеный звук оригинала, т.е. я выключаю звук оригинала, оставляя только перевод, и периодически он сам включается

**Comment by Oleg-Chashko** [comment]
> и еще маленькая проблемка, каждые 1-2 минуты, повышается виключеный звук оригинала, т.е. я выключаю звук оригинала, оставляя только перевод, и периодически он сам включается

Confirmed, this bug is happening.

**Reply by underoakerr** [reply]
Опять перестало работать(( подскажите что делать? 
![image](https://github.com/ilyhalight/voice-over-translation/assets/133874483/99bcbca9-dd48-45d4-a3b0-01bd8e381130)

**Comment by ilyhalight** [comment]
> не переводит ничего, и висит эта ошибка failed to request video translation

Попробуйте поставить cloudflare версию.
![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/abba51b9-36c4-45dc-af0b-8e1f5b3de5cb)

**Reply by underoakerr** [reply]
проблему решил включением VPN, ибо я с Украины, походу в этом и была причина...
